scripts/install-du.ps1

```diff
@@ -131,39 +175,28 @@ function Register-Components {
     $adu_extensions_dir = "$DataPath/extensions"
     $adu_extensions_sources_dir = "$adu_extensions_dir/sources"
 
-    $do_content_downloader_file = "$adu_extensions_sources_dir/deliveryoptimization_content_downloader.dll"
-    $microsoft_script_1_handler_file = "$adu_extensions_sources_dir/microsoft_script_1.dll"
-    $microsoft_simulator_1_file = "$adu_extensions_sources_dir/microsoft_simulator_1.dll"
-    $microsoft_steps_1_file = "$adu_extensions_sources_dir/microsoft_steps_1.dll"
-    $microsoft_wim_1_handler_file = "$adu_extensions_sources_dir/microsoft_wim_1.dll"
-
-    if ($TargetArch -eq "arm64") {
-        $adu_extensions_update_content_handlers_dir = "$adu_extensions_dir/update_content_handlers"
-
-        WriteExtensionRegistration -BinaryFilePath $do_content_downloader_file `
-            -OutputJsonFilePath "$adu_extensions_dir/content_downloader/extension.json"
 
-        $chName = 'content_handler.json'
-        WriteExtensionRegistration -BinaryFilePath $microsoft_script_1_handler_file `
-            -OutputJsonFilePath "$adu_extensions_update_content_handlers_dir/microsoft_script_1/$chName"
+    # Cross-compiled target means we cannot use agent binary to create the registrations, so create them in script.
```

Comment:
interesting -- my suggestion above (have the handler output json wont work on cross-compilation as we wont be able to run the binaries natively.

Maybe there's some alternative way? (have a registration template with replacable parmaeters that agnet and this code uses?)

Reply:
I wasn't thrilled about having to register components locally in order to package them, so if we can fix this up that would be great.

Reply:
I like registration template approach. Will work on that.